src/fillTank.test.js

```diff
@@ -1,11 +1,113 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
   it('should ', () => {
 
   });
 
-  // write tests here
+  it(`should not change fuelRemains if user has no money`, () => {
+    const customer1 = {
+      money: 0,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 0,
+      },
+    };
+
+    fillTank(customer1, 10, 5);
+
+    expect(customer1.vehicle.fuelRemains).toBe(0);
+  });
+
+  it(`User.money should not change more`
+    + ` than the cost of filling up to maxTankCapacity`, () => {
+    const customer1 = {
+      money: 50,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 20,
+      },
+    };
+
+    fillTank(customer1, 1, 40);
+
+    expect(customer1.money).toBe(30);
+  });
+
+  it(`should change fuelRemains to value that user can`
+  +` afford and fuelRemains <= maxTankCapacity`, () => {
+    const customer1 = {
+      money: 100, 
+      vehicle: {
+        maxTankCapacity: 40, 
+        fuelRemains: 0,
+      },
+    };
+  
+    fillTank(customer1, 1);
```

Comment:
The `fillTank` function is called without the `amount` parameter, which is correct for testing the scenario where a full tank is ordered. However, ensure that the test description clearly states this scenario.